metrics/firebase/test/firestore/rules/projects-test.js

```diff
@@ -0,0 +1,39 @@
+const {
+  setupTestDatabaseWith,
+  getApplicationWith,
+  tearDown,
+} = require("./test_utils/helpers");
+const { assertFails, assertSucceeds } = require("@firebase/testing");
+const { projects, user } = require("./test_utils/test-data");
+
+describe("Project collection rules", async function () {
+  const app = await getApplicationWith(user);
+
+  before(async () => {
+    await setupTestDatabaseWith(projects);
+  });
+
+  it("allows reading projects by an authenticated user", async () => {
+    await assertSucceeds(app.collection("projects").get());
+  });
+
+  it("allows adding projects by an authenticated user", async () => {
+    await assertSucceeds(
+      app.collection("projects").add({ name: "test_project" })
+    );
+  });
+
+  it("does not allow to read projects by an unauthenticated user", async () => {
```

Comment:
What about checking that it's not allowing to create projects by not authenticated users?
Also wouldn't it be great to add "update/delete" tests?